website_sale_attribute_filter_category/hooks.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import SUPERUSER_ID
+from odoo.api import Environment
+
+
+def post_init_hook(cr, registry):
+    """ Disable default Product Attribute Filter View """
+    env = Environment(cr, SUPERUSER_ID, {})
+    env.ref('website_sale.products_attributes').active = False
```

Comment:
This is not legit. What happen with views inheriting this one? You have to add a children view to this one with your changes and with priority="999"

Reply:
Inherit this view is complicated... but i will try to do it